apps/web/src/lang/fr.json

```diff
@@ -3572,6 +3872,10 @@
     "defaultMessage": "Découvrez-en davantage sur le Programme d’apprentissage en TI pour les personnes autochtones du gouvernement du Canada",
     "description": "Heading for the Learn more dialog"
   },
+  "LAW/ov": {
+    "defaultMessage": "Facteurs de propriété intellectuelle (par exemple, les logiciels propriétaires.)",
```

Comment:
Well, it's not consistent between English and French as it stands. The English string uses the latin abbreviation e.g. whereas the French spells it out completely. [This Government of Canada style guide](https://design.canada.ca/style-guide/#wp4-4-1) says to avoid using latin abbreviations altogether. What about changing the English string to use for example instead of e.g.?